packages/web-components/src/avatar/avatar.styles.ts

```diff
@@ -0,0 +1,569 @@
+  colorBrandBackgroundStatic,
+  colorNeutralBackground1,
+  colorNeutralBackground6,
+  colorNeutralForeground3,
+  colorNeutralForegroundInverted,
+  colorPaletteAnchorBackground2,
+  colorPaletteAnchorForeground2,
+  colorPaletteBeigeBackground2,
+  colorPaletteBeigeForeground2,
+  colorPaletteBlueBackground2,
+  colorPaletteBlueForeground2,
+  colorPaletteBrassBackground2,
+  colorPaletteBrassForeground2,
+  colorPaletteBrownBackground2,
+  colorPaletteBrownForeground2,
+  colorPaletteCornflowerBackground2,
+  colorPaletteCornflowerForeground2,
+  colorPaletteCranberryBackground2,
+  colorPaletteCranberryForeground2,
+  colorPaletteDarkGreenBackground2,
+  colorPaletteDarkGreenForeground2,
+  colorPaletteDarkRedBackground2,
+  colorPaletteDarkRedForeground2,
+  colorPaletteForestBackground2,
+  colorPaletteForestForeground2,
+  colorPaletteGoldBackground2,
+  colorPaletteGoldForeground2,
+  colorPaletteGrapeBackground2,
+  colorPaletteGrapeForeground2,
+  colorPaletteLavenderBackground2,
+  colorPaletteLavenderForeground2,
+  colorPaletteLightTealBackground2,
+  colorPaletteLightTealForeground2,
+  colorPaletteLilacBackground2,
+  colorPaletteLilacForeground2,
+  colorPaletteMagentaBackground2,
+  colorPaletteMagentaForeground2,
+  colorPaletteMarigoldBackground2,
+  colorPaletteMarigoldForeground2,
+  colorPaletteMinkBackground2,
+  colorPaletteMinkForeground2,
+  colorPaletteNavyBackground2,
+  colorPaletteNavyForeground2,
+  colorPalettePeachBackground2,
+  colorPalettePeachForeground2,
+  colorPalettePinkBackground2,
+  colorPalettePinkForeground2,
+  colorPalettePlatinumBackground2,
+  colorPalettePlatinumForeground2,
+  colorPalettePlumBackground2,
+  colorPalettePlumForeground2,
+  colorPalettePumpkinBackground2,
+  colorPalettePumpkinForeground2,
+  colorPalettePurpleBackground2,
+  colorPalettePurpleForeground2,
+  colorPaletteRedBackground2,
+  colorPaletteRedForeground2,
+  colorPaletteRoyalBlueBackground2,
+  colorPaletteRoyalBlueForeground2,
+  colorPaletteSeafoamBackground2,
+  colorPaletteSeafoamForeground2,
+  colorPaletteSteelBackground2,
+  colorPaletteSteelForeground2,
+  colorPaletteTealBackground2,
+  colorPaletteTealForeground2,
+  fontFamilyBase,
+  fontSizeBase100,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontSizeBase500,
+  fontSizeBase600,
+  fontWeightRegular,
+  fontWeightSemibold,
+  shadow16,
+  shadow28,
+  shadow4,
+  shadow8,
+  strokeWidthThick,
+  strokeWidthThicker,
+  strokeWidthThickest,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+// TODO: Should all animation constants go to tokens/theme or globals?
+// https://github.com/microsoft/fluentui/issues/16372#issuecomment-778240665
+
+const animationDuration = {
+  duration50: '50ms',
+  duration100: '100ms',
+  duration150: '150ms',
+  duration200: '200ms',
+  duration300: '300ms',
+  duration400: '400ms',
+  duration500: '500ms',
+};
+
+const animationTiming = {
+  ultraFast: animationDuration.duration50,
```

Comment:
Do we want thee here if they aren't being used?

Reply:
Currently yes